### DISCUSSION TITLE
Modal with 3.3.7

### DISCUSSION THREAD

**Discussion by gbeukes** [twbs/bootstrap#33976] [Q&A]
Hi, I have been using the modal for a while with no issues, now all of a sudden when I call .modal('show') or .modal('toggle') on a specific modal, they all seem to open if I have more than 1 on the page. Any ideas of why this is happening?

**Comment by PF94** [comment]
My guess is that all of the modals are sharing the same name, so when you want to open one modal, it's opening all of the other modals.

If there's a reason for using Bootstrap 3 in 2021 (Bootstrap 3 came out nearly 8 years ago), I'd recommend using 3.4.1 instead of 3.3.7 as there's apparantly a XSS issue with modals on 3.3.7 that got patched in 3.4.0.

**Reply by Sameu088** [reply]
Through the [faceitfinder](https://faceitaccount.com/), players can check their faceit stats within seconds.

**Comment by gbeukes** [comment]
Hi,

Thanks, I will be changing the version soon, and no the modals all have different ids, so when I call the show or hide, it is only on the specific modal. That said, even SummerNote, which does the same thing, but I will try the newer version. The only reason I have not yet switched to 4 is there are some changes that I need to take care of first and I am on a rather tight schedule.

**Comment by gbeukes** [comment]
![Summer](https://user-images.githubusercontent.com/447200/118275454-6e159600-b494-11eb-8cba-f233ae440d9d.jpg)
Here is a screenshot of summer note. Here I clicked on the help icon, but note behind it you can see all the other modals are open. If I click on the add picture for example, it still opens the others, with the help on top. I am only using this as an example, even my own pages with 2 modals always opens them both.

**Reply by PF94** [reply]
Doesn't happen on my side (on the official website, maybe your issue only happens on your possible fork of the project?)
![image](https://user-images.githubusercontent.com/45898787/118344063-55d86200-b4fa-11eb-8175-b772db80fde8.png)

What is the version of your browser?

**Comment by gbeukes** [comment]
Chrome 90.0.4430.212, also tried it on Edge 90.0.818.62, Firefox 88.0.1. I even tried it on IE 11. All are doing the same thing. As for Bootstrap, is was installed into application using Libman and Visual Studio.  For the life of me I cannot figure out why it is behaving this way.

**Comment by freegold** [comment]
Hello new to bootstrap and using softaculous to populate bootstrap in a directory. It's now in its own dir, but in php not html. So, I went to the beginning information page and didn't find any instruction as to what to do. I then downloaded 3.3.7 from the information page and unzipped it in the bootstrap dir but the index file doesn't find the unzipped folder, what do I need to do? And should I find and download 3.4.1 also? My server is CWP using Apache: Apache/2.4.56 PHP: 7.4.33 MySQL: 10.4.30-MariaDB if this helps. thanks
